src/app/models/session.rb

```diff
@@ -116,6 +116,29 @@ def recommended_sessions
     end
   end
 
+  def attendance_count
+    @attendance_count ||= attendances.count
+  end
+
+  attr_writer :attendance_count
```

Comment:
It would be helpful to say that we use this in `preload_attendance_counts` as an optimization.